source/_integrations/yolink.markdown

```diff
@@ -24,8 +24,15 @@ Integrates [YoLink](https://www.yosmart.com/) Devices into Home Assistant.
 
 {% include integrations/config_flow.md %}
 
-Unless configured otherwise, Home Assistant will use account linking provided by
-Nabu Casa for authenticating with YoLink, If you want to use separate credentials, please contact <service@yosmart.com> to obtain a `client_id` and `client_secret`. Then you can add your credentials via application credentials. Settings > Devices & Services > click the menu (three dots at the top right of the screen) and then **Application Credentials**. Enter your credentials in the pop-up window.
+<br>
+
+<div class='note'>
+
+Because it uses the Home Assistant Cloud account linking service you **must** have either `cloud:` or `default_config:` in your `configuration.yaml`. These follow steps are not required, nor will they be supported if issues are encountered.
+
+</div>
```

Comment:
This is not something that belongs on this page. If a user actively decides to remove the `cloud` dependency, that is their call. This integration does not require its use.

> nor will they be supported if issues are encountered.

They are supported? The account linking service is an additional but not required service. It is just as much supported as anything else.

Reply:
Since our customer didn't know we used Cloud Account Linking, they sent us email asking client_id and client_secret, I would like to add an obvious hint